### DISCUSSION TITLE
hostname/nickname for Meshcentral (echo TRMM to Meshcentral)

### DISCUSSION THREAD

**Discussion by marlo1973** [amidaware/tacticalrmm#1708] [Ideas]
We have several computer and server names for different customers such as "SRV2016", "SRV2019", "PC01", "PC02". If we now start MeshCentralRouter.exe, we get an overview with, for example, 3x "SRV2016", 6x "PC01" and cannot distinguish which customer is affected. Of course our naming isn't very creative, but unfortunately that's how it is now. OK, we can rename the hostname of a PC, but renaming a server is not always possible. A “Nichname” field would be sensational! Example: When the Nickname field is served, the nickname is passed to Meshcentral. If not, the actual hostname applies.

How do you do the identification of the systems?

Many greetings to all Tactical RMM supporters.

**Comment by ssteeltm** [comment]
Here I use mesh tag file to rename

**Comment by marlo1973** [comment]
Hi ssteeltm, yes ok, I can create a tag for the customer in the web interface. I can then display an overview of the customer in the web interface, that's right. But unfortunately I can't work with the tags in MeshCentralRouter.exe. I think MeshCentralRouter.exe is absolutely brilliant. The tag option is missing there.

**Comment by ssteeltm** [comment]
I use it to rename device, "company a - computer1" , so this way it show right on router

**Comment by marlo1973** [comment]
Ahh, you mean if I expand the config.json with the following entry:

"agenttag": {
      "ServerName": 1,
      "ServerDesc": 1,
      "ServerTags": 1
    }

and then the MeshAgent.tag file with content in the program directory?
~ServerName: Test_Servername
~ServerDesc: Test_Serverdesc
~ServerTags: Test_Servertags

Unfortunately nothing happens for me at all... My config.json looks like this:
{
  "settings": {
    "Cert": "mesh.xxx.com",
    "MongoDb": "mongodb://127.0.0.1:27017",
    "MongoDbName": "meshcentral",
    "WANonly": true,
    "Minify": 1,
    "Port": 4430,
    "AliasPort": 443,
    "RedirPort": 800,
    "AllowLoginToken": true,
    "AllowFraming": true,
    "_AgentPing": 60,
    "AgentPong": 300,
    "AllowHighQualityDesktop": true,
    "TlsOffload": "127.0.0.1",
    "agentCoreDump": false,
    "Compression": true,
    "WsCompression": true,
    "AgentWsCompression": true,
    "MaxInvalidLogin": { "time": 5, "count": 5, "coolofftime": 30 }
  },
  "domains": {
	"": {
      "Title": "Tactical RMM",
      "Title2": "Tactical RMM",
      "NewAccounts": false,
      "CertUrl": "https://mesh.xxx.com:443/",
	  "GeoLocation": true,
      "CookieIpCheck": false,
	  "IgnoreAgentHashCheck": true,
      "mstsc": true
    },
    "agenttag": {
      "ServerName": 1,
      "ServerDesc": 1,
      "ServerTags": 1
    }
  }
}

Is my config.json wrong?

**Comment by marlo1973** [comment]
ok, found my mistake.
"domains": {
	"": {
    "agenttag": {
        "ServerName": 1,
        "ServerDesc": 1,
        "ServerTags": 1
      },
      "Title": "Tactical RMM",
      "Title2": "Tactical RMM",
      "NewAccounts": false,
      "CertUrl": "https://mesh.xxx.com:443/",
	  "GeoLocation": true,
      "CookieIpCheck": false,
	  "IgnoreAgentHashCheck": true,
      "mstsc": true
    }
  }

my config.json was incorrect. Now it's working. Thank you!
